libraries/core_libs/network/src/tarcap/packets_handlers/status_packet_handler.cpp

```diff
@@ -68,7 +68,8 @@ void StatusPacketHandler::process(const PacketData& packet_data, const std::shar
       if (pbft_synced_period + node_history < peer_pbft_chain_size) {
         LOG(log_nf_) << "Light node is not able to serve our syncing request. " << packet_data.from_node_id_.abridged()
                      << " peer will be disconnected";
-        disconnect(peer->getId(), dev::p2p::UserReason);
+        disconnect(packet_data.from_node_id_, dev::p2p::UserReason);
```

Comment:
What is the difference between "peer->getId()" and "packet_data.from_node_id_"? Are they same thing?

Reply:
peer can be null here, if peer is not null it is the same value, but this is safer